pmm-app/src/pmm-dbaas/components/XtraDB/AddXtraDBModal/XtraDBAdvancedOptions/XtraDBAdvancedOptions.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { stylesFactory } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
```

Comment:
We need no specific styles for the light theme, right?

Reply:
Yes we don't need. No colors being used here, just the spacing from the theme.